selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/CheckoutToRemoteBranchTest.java

```diff
@@ -10,273 +10,143 @@
-
-    // check the change into git compare form
-    git.clickCompareBtnGitHistory();
-    git.waitGroupGitCompareIsOpen();
-    git.selectFileInChangedFilesTreePanel("GreetingController.java");
-    checkChangesIntoCompareForm(uniqueValue);
-    seleniumWebDriver.switchTo().parentFrame();
-    git.closeGitCompareForm();
-    git.waitGroupGitCompareIsOpen();
-    git.selectFileInChangedFilesTreePanel("index.jsp");
-    checkChangesIntoCompareForm(uniqueValue2);
-  }
+    checkoutToSecondRemoteBranch();
 
-  private void importProjectFromRemoteRepo(String urlRepo, String projectName) {
-    importFromLocation.waitAndTypeImporterAsGitInfo(urlRepo, projectName);
-    projectWizard.waitCreateProjectWizardForm();
-    projectWizard.selectTypeProject(Wizard.TypeProject.MAVEN);
-    loader.waitOnClosed();
-    projectWizard.clickSaveButton();
-    loader.waitOnClosed();
-    projectWizard.waitCreateProjectWizardFormIsClosed();
-    projectExplorer.waitItem(projectName);
-    loader.waitOnClosed();
+    // check the changes are present in the files
+    projectExplorer.quickExpandWithJavaScript();
+    projectExplorer.openItemByPath(PROJECT_NAME2 + "/src/main/java/che/eclipse/sample/Aclass.java");
+    editor.waitTextIntoEditor(changeContent);
+    projectExplorer.openItemByPath(PROJECT_NAME2 + "/src/main/webapp/index.jsp");
+    editor.waitTextIntoEditor(changeContent);
   }
 
   private void checkoutToSecondRemoteBranch() throws Exception {
-    loader.waitOnClosed();
-    menu.runCommand(TestMenuCommandsConstants.Git.GIT, TestMenuCommandsConstants.Git.BRANCHES);
+    menu.runCommand(GIT, BRANCHES);
     git.waitBranchInTheList(MASTER_BRANCH);
     git.waitBranchInTheList(ORIGIN_MASTER_BRANCH);
     git.waitBranchInTheList(ORIGIN_SECOND_BRANCH);
     git.selectBranchAndClickCheckoutBtn(ORIGIN_SECOND_BRANCH);
-    loader.waitOnClosed();
-    menu.runCommand(TestMenuCommandsConstants.Git.GIT, TestMenuCommandsConstants.Git.BRANCHES);
+    git.waitGitCompareBranchFormIsClosed();
+    menu.runCommand(GIT, BRANCHES);
     git.waitBranchInTheListWithCoState(SECOND_BRANCH);
     git.closeBranchesForm();
   }
 
-  private void makePullSecondRemoteBranch() throws InterruptedException {
+  private void performGitPull() throws InterruptedException {
+    menu.runCommand(GIT, REMOTES_TOP, PULL);
     git.waitPullFormToOpen();
     git.waitPullRemoteRepository(NAME_REMOTE_REPO);
     git.waitPullRemoteBranchName(SECOND_BRANCH);
     git.waitPullLocalBranchName(SECOND_BRANCH);
     git.clickPull();
     git.waitPullFormToClose();
+    git.waitGitStatusBarWithMess(PULL_MSG);
   }
 
-  private void checkChangesIntoCompareForm(String expText) {
-    git.clickOnGroupCompareButton();
-    git.waitGitCompareFormIsOpen();
-    git.waitExpTextIntoCompareLeftEditor(expText);
-    git.waitTextNotPresentIntoCompareRightEditor(expText);
+  private void changeContentOnGithubSide(String pathToContent, String content, String branchName)
+      throws IOException {
+    testRepo
```

Comment:
It is better to encapsulate file update into separate method `update(Path relativePathToFile, byte[] newContent, String branchName)` in _TestGitHubRepository_ class.